test/cbmc/proofs/parsing/ProcessReceivedTCPPacket_IPv6/Makefile.json

```diff
@@ -0,0 +1,41 @@
+{
+  "ENTRY": "ProcessReceivedTCPPacket_IPv6",
+  "CBMCFLAGS":
+  [
+    "--unwind 1",
+    "--unwindset prvTCPSendRepeated.0:13",
+    "--nondet-static"
+  ],
+  "OBJS":
+  [
+    "$(ENTRY)_harness.goto",
+    "$(FREERTOS_PLUS_TCP)/test/cbmc/stubs/cbmc.goto",
+    "$(FREERTOS_PLUS_TCP)/source/FreeRTOS_IP.goto",
+    "$(FREERTOS_PLUS_TCP)/source/FreeRTOS_IP_Utils.goto",
+    "$(FREERTOS_PLUS_TCP)/source/FreeRTOS_IP_Timers.goto",
```

Comment:
I doubt we need all these includes, can we also mention the functions that are being validated as part of each test in the commit message? It will give idea of which includes are really needed.

Reply:
Remove redundant goto files. And update description of test case for sub-functions also validated.